mw-webapp/src/logic/aboutProjectPage/AboutProjectPage.module.scss

```diff
@@ -1,18 +1,114 @@
 @import "src/styles/variables";
 
+$maxWidth: 1200px;
+
 .pageWrapper {
   display: flex;
   flex-direction: column;
   align-items: center;
   padding: 0 $gapMedium;
   margin-top: $marginMedium;
+  gap: $gapBig;
+}
+
+.title {
+  color: var(--secondaryTextButtonColor);
+  text-align: center;
+}
+
+.subtitle {
+  color: var(--primaryTextColor);
+  text-align: center;
+  margin-bottom: $marginSmall;
+}
+
+.aboutBlock {
+  max-width: $maxWidth;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: $gapSmall;
+  margin-top: $marginBig;
```

Comment:
@Ekaterina1994 do we use margins to separate blocks?

Reply:
fixed

Reply:
paddings to separate blocks? it's really strange, need you detailed comment @Ekaterina1994 . Also @AKiroi tell me why are you agree with such approach?

Reply:
I don't agry to separate blocks with padding, its better to do with margin or gap

Reply:
Removed this padding

Reply:
@tritonJS826  answered in personal messages